<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1159</width>
    <height>896</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1159</width>
    <height>896</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1159</width>
    <height>896</height>
   </size>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="card3">
   <property name="geometry">
    <rect>
     <x>294</x>
     <y>626</y>
     <width>111</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="card1">
   <property name="geometry">
    <rect>
     <x>454</x>
     <y>626</y>
     <width>111</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="card2">
   <property name="geometry">
    <rect>
     <x>624</x>
     <y>626</y>
     <width>111</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="card6">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>626</y>
     <width>111</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="card4">
   <property name="geometry">
    <rect>
     <x>794</x>
     <y>626</y>
     <width>111</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="card5">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>630</y>
     <width>111</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="card11">
   <property name="geometry">
    <rect>
     <x>194</x>
     <y>446</y>
     <width>111</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="card9">
   <property name="geometry">
    <rect>
     <x>364</x>
     <y>448</y>
     <width>111</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="card7">
   <property name="geometry">
    <rect>
     <x>524</x>
     <y>446</y>
     <width>101</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="card8">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>448</y>
     <width>111</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="card10">
   <property name="geometry">
    <rect>
     <x>824</x>
     <y>448</y>
     <width>121</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="give_but">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>790</y>
     <width>75</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Взять</string>
   </property>
  </widget>
  <widget class="QPushButton" name="skip_but">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>790</y>
     <width>75</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Всё</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>0</y>
     <width>91</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Карты дилера</string>
   </property>
  </widget>
  <widget class="QLabel" name="botcard1">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>50</y>
     <width>111</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="botcard2">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>52</y>
     <width>111</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="botcard4">
   <property name="geometry">
    <rect>
     <x>726</x>
     <y>52</y>
     <width>111</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="botcard3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>111</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="botcard6">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>111</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="botcard5">
   <property name="geometry">
    <rect>
     <x>890</x>
     <y>52</y>
     <width>111</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="botcard7">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>230</y>
     <width>111</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="botcard8">
   <property name="geometry">
    <rect>
     <x>566</x>
     <y>232</y>
     <width>111</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="back_but">
   <property name="geometry">
    <rect>
     <x>1074</x>
     <y>10</y>
     <width>71</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>В меню</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>870</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Ставка:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="up50">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>790</y>
     <width>75</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>+50</string>
   </property>
  </widget>
  <widget class="QPushButton" name="up100">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>790</y>
     <width>75</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>+100</string>
   </property>
  </widget>
  <widget class="QPushButton" name="up500">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>790</y>
     <width>75</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>+500</string>
   </property>
  </widget>
  <widget class="QPushButton" name="up1000">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>790</y>
     <width>75</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>+1000</string>
   </property>
  </widget>
  <widget class="QPushButton" name="down50">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>790</y>
     <width>75</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>-50</string>
   </property>
  </widget>
  <widget class="QPushButton" name="down100">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>790</y>
     <width>75</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>-100</string>
   </property>
  </widget>
  <widget class="QPushButton" name="down500">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>790</y>
     <width>75</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>-500</string>
   </property>
  </widget>
  <widget class="QPushButton" name="down1000">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>790</y>
     <width>75</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>-1000</string>
   </property>
  </widget>
  <widget class="QLabel" name="bet">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>870</y>
     <width>171</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>870</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Баланс:</string>
   </property>
  </widget>
  <widget class="QLabel" name="balance">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>870</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>870</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Очки:</string>
   </property>
  </widget>
  <widget class="QLabel" name="score">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>870</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
